src/fonduer/learning/disc_learning.py

```diff
@@ -86,50 +88,63 @@ def train(
         dev_ckpt_delay=0.75,
         save_dir="checkpoints",
         seed=1234,
-        **kwargs
+        host_device="CPU",
     ):
         """
         Generic training procedure for PyTorch model
 
-        :param X_train: The training Candidates which is a pair with a list of
-            Candidate objects and a csr_AnnotationMatrix with rows corresponding
+        :param X_train: The training Candidates with a pair with a list of
```

Comment:
Can we make rephrase this parameter description a bit? I can't really understand what is described here

Reply:
Done!